runtime/tutor.txt

```diff
@@ -214,7 +214,8 @@ _________________________________________________________________
  whitespace, whereas words can be separated by other characters
  in addition to whitespace.
 
- All of these motions select the text they traverse.
+ All of these motions select the final word and whitespace they
+ traverse.
```

Comment:
I'm not really sure how this makes things clearer? Also, if two lines are added, two lines need to be removed to maintain the spacing, so an extra blank line should be removed.

Reply:
What you said about the spacing makes sense. I can adjust that.

The reason it is clearer (to me) is that, for example, if a cursor is somewhere on word A in a sequence of words A B C D, and I enter a motion `3w` in normal mode, the text between where the cursor was before I entered `3w` and where it is after, which is the end of the whitespace following word C, _is not _selected._ Rather, the text selected is only the last  word and and the whitespace between words C and D. Similarly for all of `w`,`e`,`b` / `W`,`E`,`B`.

I think having a behavior to select _all_ the text from the cursor (or the word on which the cursor is located) to the destination of the motion would be more useful in general to a user, but as it stands, the behavior is not this, so we should adjust the tutor to reflect this, IMO.

Reply:
The reason only the last word is selected is because the motion is being repeated, and each repetition overrides the old selection, causing the final repetition to be the only one whose selection is left. Additionally, I don't think this behaviour is really that clear from what is written here; and keep in mind that this isn't a formal definition, it's meant to introduce new users; so rewording the description like this would really mostly serve to confuse the user, I think.

Reply:
Is there a way to 'select the next 3 words' or just 'select the next word, three times'?

To me this is a non-trivial distinction, and it might be a good idea to include such notions in the introduction.